query/src/main/scala/filodb/query/exec/rangefn/AggrOverTimeFunctions.scala

```diff
@@ -784,6 +852,27 @@ class QuantileOverTimeChunkedFunctionD(funcParams: Seq[FuncArgs]) extends Quanti
   }
 }
 
+class MedianAbsoluteDeviationOverTimeChunkedFunctionD
```

Comment:
Same comments as `MedianAbsoluteDeviationOverTimeChunkedFunctionL` please clean up this implementation accordingly.